src/subscriptions.ts

```diff
@@ -348,18 +347,11 @@ function indexerPreFilter(
     if (subscription.notePrefix) {
       filter = filter.notePrefix(Buffer.from(subscription.notePrefix).toString('base64'))
     }
-    if (
-      subscription.appId &&
-      (typeof subscription.appId === 'number' || (typeof subscription.appId === 'bigint' && subscription.appId <= Number.MAX_SAFE_INTEGER))
-    ) {
-      filter = filter.applicationID(Number(subscription.appId))
+    if (subscription.appId && typeof subscription.appId === 'bigint' && subscription.appId <= Number.MAX_SAFE_INTEGER) {
```

Comment:
Do we still need the MAX_SAFE_INTEGER stuff? ditto for other places in codebase...